libs/langchain/tests/unit_tests/llms/test_aleph_alpha.py

```diff
@@ -0,0 +1,32 @@
+"""Test Aleph Alpha specific stuff."""
+
+from pytest import CaptureFixture, MonkeyPatch
+
+from langchain.llms.aleph_alpha import AlephAlpha
+from langchain.pydantic_v1 import SecretStr
+
+
```

Comment:
Could you skip these tests if aleph-alpha is not installed? Most langchain dependencies are optional. 

Two options: 

1) Add the dependency to extended testing extra in pyproject.toml , and then use the langchain custom @pytest.mark.requires marker. This marker will skip it in regular unit tests, but run the tests on CI in extended unit tests where this dependency will be installed
2) Use pytest skip to skip running the entire module if the alpeh alpha package is missing

Reply:
Sure. I'll implement the first option later today.